cl/corpus_importer/tests.py

```diff
@@ -2387,3 +2402,497 @@ def test_find_and_fix_mis_matched_dockets(self, mock_download_ia):
+            After the disposal of counts 2 and 3, the only charge before the court and jury was that the defendant had sold distilled spirits and alcohol as a retail dealer, without first obtaining a license from the State Revenue Commissioner. The evidence adduced to show the guilt of the accused on count 1 was wholly circumstantial, and was insufficient to exclude every reasonable hypothesis except that of his guilt, and it failed to show beyond a reasonable doubt that he had sold distilled spirits or alcohol. The cases of <em>Thomas</em> v. <em>State,</em> <cross_reference><span class="citation no-link">65 Ga. App. 749</span></cross_reference> (<cross_reference><span class="citation" data-id="3407553"><a href="/opinion/3412403/thomas-v-state/">16 S.E.2d 447</a></span></cross_reference>), and <em>Martin</em> v. <em>State,</em> <cross_reference><span class="citation no-link">68 Ga. App. 169</span></cross_reference> (<cross_reference><span class="citation" data-id="3405716"><a href="/opinion/3410794/martin-v-state/">22 S.E.2d 193</a></span></cross_reference>), cited in behalf of the defendant in error, are distinguished by their facts from this case. The verdict was contrary to law and the evidence; and the overruling of the certiorari was error.</p>
+            <p><em>Judgment reversed. MacIntyre, J., concurs.</em></p>"""
+        concurrence = """<p>Under the record the judgment should be reversed for another reason. Since the jury, based on the same evidence, found the defendant not guilty on count 2 for possessing liquors, and a verdict finding him guilty on count 1 for selling intoxicating liquors, the verdicts are repugnant and void as being inconsistent verdicts by the same jury based on the same evidence. <em>Britt</em> v. <em>State,</em> <cross_reference><span class="citation no-link">36 Ga. App. 668</span></cross_reference>
+            (<cross_reference><span class="citation no-link">137 S.E. 791</span></cross_reference>), and cit.; <em>Kuck</em> v. <em>State,</em> <cross_reference><span class="citation" data-id="5582722"><a href="/opinion/5732248/kuck-v-state/">149 Ga. 191</a></span></cross_reference>
+            (<cross_reference><span class="citation no-link">99 S.E. 622</span></cross_reference>). I concur in the reversal for this additional reason.</p>"""
+
+        cluster = OpinionClusterFactoryMultipleOpinions(
+            docket=DocketFactory(source=Docket.COLUMBIA),
+            sub_opinions__data=[
+                {"type": "020lead", "html_with_citations": lead},
+                {"type": "030concurrence", "html_with_citations": concurrence},
+            ],
+        )
+
+        self.assertEqual(
+            OpinionCluster.objects.get(id=cluster.id).attorneys, "", msg="WHAT"
```

Comment:
WHAT?

Reply:
A thought I just had reviewing this and other debug statements that might be useful is that if instead of doing `WHAT` or `12345` or some other random thing, it might be smart to always use your initials or `TODO`. Then at the end of a PR, it'll be easy to track down any strays. If you use TODO, PyCharm even has a feature to help find those markers. A thought.

Reply:
I corrected the test and tried to put a more informative message